src/eduid/webapp/idp/other_device/data.py

```diff
@@ -0,0 +1,17 @@
+"""
+Some data structures that causes import loops if they are defined in db.py.
+"""
+from enum import Enum
+from typing import NewType
+
+
+class OtherDeviceState(str, Enum):
+    NEW = 'NEW'  # only used device #1 so far
+    IN_PROGRESS = 'IN_PROGRESS'  # device #2 has 'grabbed' the request
+    LOGGED_IN = 'LOGGED_IN'  # device #2 is finished with the request (successfully)
```

Comment:
You talked about renaming LOGGED_IN to AUTHENTICATED, right?

Reply:
done